.scalafix.conf

```diff
@@ -1 +1,7 @@
-OrganizeImports.preset = INTELLIJ_2020_3
+rules = [OrganizeImports]
+
+OrganizeImports {
+  preset = INTELLIJ_2020_3
+  removeUnused = false # `true` is not supported in Scala 3.3.0
```

Comment:
Do you think we should worry about Scala 3 support while using 
```
scalaVersion := "2.13.14"
```
?

Reply:
I found using 
```
removeUnused = true
```
pretty useful, IMO

Reply:
`scalac` complains about that too, but I returned it